src/components/TaskRunDetails.vue

```diff
@@ -12,6 +12,14 @@
       </template>
     </p-key-value>
 
+    <template v-if="result">
+      <p-key-value :label="localization.info.result" :alternate="alternate">
+        <template v-if="result.description" #value>
+          <p-markdown-renderer v-if="result.description" :text="result.description" class="task-run-details__markdown-renderer" />
```

Comment:
Should the `v-if="result.description"` just be combined with the top level <template> check? I'd imagine you wouldn't care to show the label without the description.

Also looks like there's multiple checks for the description, maybe Typescript isn't clever enough to know it's already been checked for in the containing `<template #value>`?

Reply:
Usually a v-if on the `#value` ref of a `p-key-value` means that the `None` should show up.

Reply:
Yup @pleek91 is correct here - that's potentially not super clear from reading the markup but it's how the `PKeyValue` component works (and how we use it elsewhere)